shell/promptRemove/mixin/roleDeletionCheck.js

```diff
@@ -66,12 +66,26 @@ export default {
           method:        'get',
         }, { root: true });
 
+        // We need to fetch the users here in order to get an accurate count when selecting global roles.
+        const users = await this.$store.dispatch('management/request', {
+          url:           `/v1/${ MANAGEMENT.USER }`,
+          method:        'get',
+        }, { root: true });
+
+        const userMap = users.data?.reduce((map, user) => {
+          if ( user.username ) {
```

Comment:
I'm not 100% sure, but i think what you're after here is to exclude system accounts which could be done via `user.isSystem`? For this user case maybe the existing `user.username` is better?

Reply:
Yes exactly, system accounts was the term I was looking for. I'm not entirely sure if using `user.isSystem` would be much better than searching for the username as `user.username` only exists if it isn't a system account.

Reply:
Let's keep as is for the moment. `isSystem` was brought in by Vince to do I think the exact task as here, but given  everything is tested and working lets get this in and re-visit if there's issues

Reply:
@richard-cox Agree - I think this is okay